index.html

```diff
@@ -250,6 +250,37 @@ <h3>LET'S CONNECT</h3>
                 </div>
             </div>
         </section>
+        <section id="contactcontainer">
+            <div class="rectangular1"></div>
+            <div class="rectangular2"></div>
+            <div class="circle2"></div>
+            <div class="circle1"></div>
+            <div class="ellipse2"></div>
+            <div class="contact_content">
+                <h1 class="contact_title">Contact me</h1>
+                <p class="contact_description">
+                    If you have an application you are interested in developing, a
+                    feature that you need built or a project that needs coding. I’d
+                    love to help with it.
+                </p>
+            </div>
+            <div class="formcontainer">
+                <form id="input" action="https://formspree.io/f/xayvlqqy" method="post">
+                    <label class="label" for="username"></label>
+                    <input id="username" class="inputs" name="username" type="text" maxlength="30"
+                        placeholder="Full name" required="required" />
```

Comment:
- [ ] [Optional] For the attribute `required` you can only use `required` instead of `required="required"`